core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/util/DataTransferObject.java

```diff
@@ -0,0 +1,5 @@
+package ca.jrvs.apps.jdbc.util;
+
+public interface DataTransferObject {
+    long getId();
+}
```

Comment:
`git rm target`
please remove target directory.
all generated files(e.g. class, log, tmp, output) should not be committed to git